manufacturing-ai-suite/wind-turbine-anomaly-detection/docker-compose.yml

```diff
@@ -339,6 +334,21 @@ services:
       MINIO_SECRET_KEY: ${MR_MINIO_SECRET_KEY}
     command: server --address ":8000" --certs-dir /certs /data
 
+
+  nginx:
+    image: nginx:alpine
```

Comment:
Ideally this shouldn't be a problem as we are using this from docker hub but going by OSPDT experience they say there are few obligations to use alpine based images.

Let's use nginx debian OR ubuntu based base images.